editor/src/clj/editor/app_view.clj

```diff
@@ -873,7 +873,7 @@
 (defn- on-service-url-found [prefs target]
   (engine/apply-simulated-resolution! prefs target))
 
-(defn- launch-built-project! [project engine-descriptor project-directory prefs web-server debug?]
+(defn- launch-built-project! [project workspace engine-descriptor project-directory prefs web-server debug?]
```

Comment:
You can get the `workspace` from the `project` using `project/workspace`, so you don't need to add a `workspace` argument here.